waterlog/src/Components/AlertsMap.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Loader from './Loader';
+import MapComponent from './Map';
+
+const AlertsMap = ({ error, loading, sensorLocations }) => {
+	if (error) {
+		return <div>Error! {error.message}</div>;
```

Comment:
Choose and indent level, You cannot be changing your mind for each file

Reply:
Fixed